django/contrib/admindocs/views.py

```diff
@@ -145,13 +156,29 @@ def get_context_data(self, **kwargs):
 class ViewDetailView(BaseAdminDocsView):
     template_name = 'admin_doc/view_detail.html'
 
-    def get_context_data(self, **kwargs):
-        view = self.kwargs['view']
+    @staticmethod
+    def _get_view_func(view):
         urlconf = get_urlconf()
         if get_resolver(urlconf)._is_callback(view):
             mod, func = get_mod_func(view)
-            view_func = getattr(import_module(mod), func)
-        else:
+            try:
+                return getattr(import_module(mod), func)
+            except ImportError:
+                # Unable to import the module - try processing it again to get the class.
+                mod, klass = get_mod_func(mod)
+                return getattr(getattr(import_module(mod), klass), func)
+
+    def get_context_data(self, **kwargs):
+        view = self.kwargs['view']
+        try:
+            view_func = self._get_view_func(view)
+        except AttributeError:
```

Comment:
I think it would be clearer if `_get_view_func` did the exception handling and returned `None` for this case.